inst/extdata/OSD-error-reporting/RO/davis-hz.csv

```diff
@@ -10,6 +10,10 @@
 "BLANKOUT","Oi",3,0
 "BOOMTOWN","02",8,0
 "BOOMTOWN","0l",10,8
+"BOOMTOWN","IIC",135,NA
```

Comment:
depth issues due to now non-numeric depths returned in some cases (previously unparsed)